docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -1765,291 +1765,347 @@ The following error message codes may appear on the app-server log. Use the foll
 
 
 
-### ZWED0063W":"Stack trace follows\n%s
+### ZWED0063W
+
+  Stack Trace Exception
```

Comment:
This isn't what the log message is

Reply:
as in, literally what it says:

"Stack trace follows
_stackTrace_"

is what it should be